src/pages/en/guides/deploy/gitlab.md

```diff
@@ -13,30 +13,44 @@ Check out [the official GitLab Pages Astro example project](https://gitlab.com/p
 
 ## How to deploy
 
-1. Set the correct `.site` in `astro.config.mjs`.
-2. Set `dist` in `astro.config.mjs` to `public` and `public` in `astro.config.mjs` to a newly named folder that is holding everything currently in `public`. The reasoning is because `public` is a second source folder in astro, so if you would like to output to `public` you'll need to pull public assets from a different folder. Your `astro.config.mjs` might end up looking like this:
+1. Set the correct `site` in `astro.config.mjs`.
+2. Set `dist` in `astro.config.mjs` to `public`. GitLab Pages requires exposed files to be located in a folder called "public". So we're instructing Astro to put the static build output in a folder of that name.
+3. If you were using the [`public/` directory](en/guides/images/#public) as a source of static files, you need to rename the folder and set `public` in `astro.config.mjs` accordantly.
+
+Here is an example of a `astro.config.mjs` where the `public/` directory is set to `static`.
 
    ```js
    export default defineConfig({
      sitemap: true,
      site: 'https://astro.build/',
+     dist: 'public',
+     public: 'static',
    });
```

Comment:
`````suggestion
2. Set `dist` in `astro.config.mjs` to `public`. GitLab Pages requires exposed files to be located in a folder called "public". This instructs Astro to put the static build output in a folder called `public`.

If you were using the [`public/` directory](/en/guides/images/#public) as a source of static files in your Astro project, rename it and use that new folder name in `astro.config.mjs` for the value of `public`.

For example, here are the correct `astro.config.mjs` settings when the `public/` directory is renamed to `static/`:

   ```js
   export default defineConfig({
     sitemap: true,
     site: 'https://astro.build/',
     dist: 'public',
     public: 'static',
   });
`````
Check the preview to make sure this is properly indented, but since there is only one code sample shown for both sets of instructions (`dist` and `public`), I think it makes sense to keep them together both as step 2. (And then, the step below Step 3 doesn't need to renamed.)